api/src/db/db.service.ts

```diff
@@ -102,28 +102,39 @@ export class DbService {
     }
 
     /**
-     * Get data to which a user has access and has subscribed to
-     * @param {Array<string>} accessTags - Array with access tag ID's
-     * @param {Array<string>} subscriptionTags - Array with subscription tag ID's
+     * Get data to which a user has access to including the user document itself.
+     * @param {string} userID - User document ID.
+     * @param {Array<string>} readAcl - Array with read access acl IDs to which the user has access to. This should be an expanded list of acl IDs.
+     * @param {DbService.getDocsOptions} options - Query configuration object.
      * @returns - Promise containing the query result
      */
-    getDocs(accessTags: Array<string>, subscriptionTags: Array<string>) {
+    getDocs(
```

Comment:
Maybe add a test for this? Then we can get a better feel for how to fetch different types of documents.